build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -297,9 +299,9 @@ class WatchImpl implements BuildState {
 
   /// Checks if we should skip a watch event for this [change].
   bool _shouldProcess(AssetChange change) {
-    assert(_assetGraph != null);
-    if (_isCacheFile(change) && !_assetGraph.contains(change.id)) return false;
-    var node = _assetGraph.get(change.id);
+    assert(assetGraph != null);
```

Comment:
are these changes necessary? should we still refer to `_assetGraph` from within this class?

Reply:
done